discovery-provider/src/queries/get_top_users.py

```diff
@@ -1,23 +1,32 @@
-from sqlalchemy.sql.expression import desc
-from src.models.models import AggregateUser, User
-from src.queries.query_helpers import helpers, paginate_query, populate_user_metadata
+from src.queries.query_helpers import get_pagination_vars, populate_user_metadata
 from src.utils.db_session import get_db_read_replica
 
+sql = """
+select users.*
+from users
+join aggregate_user using (user_id)
+where
+    is_current
+    and user_id in (
+        select user_id from aggregate_user
+        where track_count > 0
+        order by follower_count desc
```

Comment:
Does this also need `user_id asc` to be deterministic?

Reply:
good callout